pkg/pxc/app/statefulset/node.go

```diff
@@ -175,7 +175,20 @@ func (c *Node) AppContainer(spec *api.PodSpec, secrets string, cr *api.PerconaXt
 			appc.Env = append(appc.Env, logEnvs...)
 		}
 	}
+	if cr.CompareVersionWith("1.9.0") >= 0 {
+		fvar := true
+		appc.EnvFrom = []corev1.EnvFromSource{
```

Comment:
should we add Envs to the logging container?

Reply:
We already have such functional for logging container https://github.com/percona/percona-xtradb-cluster-operator/blob/main/deploy/cr.yaml#L17 and  I am not sure that we need to add the same functionality but using another secret. Otherwise it can confuse users... Why env vars were added to all containers except logging :) @delgod  what do you think ?

Reply:
up to you